app/frontend/pairings/bracketTypes.ts

```diff
@@ -0,0 +1,15 @@
+export interface BracketPlayer {
```

Comment:
you can definitely do this in a followup PR if you wish, but it would be great to have the identity name and faction here as well.